pkg/prometheus/rules_test.go

```diff
@@ -130,7 +131,7 @@ func shouldReturnAtLeastOneConfigMap(t *testing.T) {
 	}
 
 	if len(configMaps) != 1 {
-		t.Fatalf("expected one ConfigMaps but got %v", len(configMaps))
+		t.Fatalf("expected three ConfigMaps but got %v", len(configMaps))
```

Comment:
Sorry, but since we reverted our change to `makeRulesConfigMaps` this should be reverted as well
```suggestion
		t.Fatalf("expected one ConfigMaps but got %v", len(configMaps))
```